cpp/src/svm/svc_impl.cuh

```diff
@@ -72,18 +72,16 @@ void svcFitX(const raft::handle_t& handle,
   {
     rmm::device_uvector<math_t> unique_labels(0, stream);
     model.n_classes = raft::label::getUniquelabels(unique_labels, labels, n_rows, stream);
-    rmm::device_async_resource_ref rmm_alloc = rmm::mr::get_current_device_resource();
-    model.unique_labels                      = (math_t*)rmm_alloc.allocate_async(
-      model.n_classes * sizeof(math_t), rmm::CUDA_ALLOCATION_ALIGNMENT, stream);
-    raft::copy(model.unique_labels, unique_labels.data(), model.n_classes, stream);
+    model.unique_labels.resize(model.n_classes * sizeof(math_t), stream);
+    raft::copy((math_t*)model.unique_labels.data(), unique_labels.data(), model.n_classes, stream);
```

Comment:
Is `unique_labels` being re-used elsewhere or can this copy be avoided by directly passing `modeul.unique_labels` to `getUniquelabels`?

Reply:
You can use `unique_labels.release()` to get an r-value `device_buffer` and avoid the copy https://github.com/rapidsai/rmm/blob/b51447393c523cc929608d84850c70a3eae08af3/include/rmm/device_uvector.hpp#L414